staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -176,13 +176,88 @@ type Histogram struct {
 	*dto.Histogram
 }
 
-// GetHistogramFromGatherer collects a metric from a gatherer implementing k8s.io/component-base/metrics.Gatherer interface.
+// HistogramVec wraps a slice of Histogram.
+// Note that each Histogram must have the same number of buckets.
```

Comment:
Can you add this check to `func (vec HistogramVec) Validate() error`?

Reply:
done.